packages/react/src/components/AccountSettings/ConfigureTOTP/types.ts

```diff
@@ -0,0 +1,10 @@
+export interface ConfigureTOTPProps {
+  /** TOTP issuer. Default to "AWSCognito" */
+  totpIssuer?: string;
+  /** TOTP username. Default to  */
+  totpUsername?: string;
+  // callback once totp is successfully set up
```

Comment:
Sorry to keep on abt comment syntaxes, but can we align on either inline or JSDoc for this interface?

Reply:
No, this is my bad on sloppiness here xD

```suggestion
  /** callback once totp is successfully set up */
```